mitmproxy/net/http/http1/assemble.py

```diff
@@ -29,13 +29,18 @@ def assemble_response_head(response):
     return b"%s\r\n%s\r\n" % (first_line, headers)
 
 
-def assemble_body(headers, body_chunks):
+def assemble_body(headers, body_chunks, trailers):
     if "chunked" in headers.get("transfer-encoding", "").lower():
         for chunk in body_chunks:
             if chunk:
                 yield b"%x\r\n%s\r\n" % (len(chunk), chunk)
-        yield b"0\r\n\r\n"
+        if trailers:
+            yield b"0\r\n%s\r\n" % (trailers,)
```

Comment:
I think this could be simplified to `yield b"0\r\n%s\r\n" % trailers`
Or is there a reason you chose the tuple-style syntax here?